### ISSUE TITLE
'fakeRegion' while sqs using localstack 

### ISSUE THREAD

**Issue by ankitapriya** [localstack/localstack#2882]
# Type of request: This is a ...

[ ] bug report
[ ] feature request

# Detailed description

I am getting this error while using sqs using localstack.

```
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in run_wsgi
localstack_main |     execute(self.server.app)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 312, in execute
localstack_main |     application_iter = app(environ, start_response)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/server.py", line 167, in __call__
localstack_main |     return backend_app(environ, start_response)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
localstack_main |     return self.wsgi_app(environ, start_response)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 2450, in wsgi_app
localstack_main |     response = self.handle_exception(e)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
localstack_main |     return cors_after_request(app.make_response(f(*args, **kwargs)))
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1867, in handle_exception
localstack_main |     reraise(exc_type, exc_value, tb)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
localstack_main |     raise value
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
localstack_main |     response = self.full_dispatch_request()
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
localstack_main |     rv = self.handle_user_exception(e)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
localstack_main |     return cors_after_request(app.make_response(f(*args, **kwargs)))
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
localstack_main |     reraise(exc_type, exc_value, tb)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
localstack_main |     raise value
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
localstack_main |     rv = self.dispatch_request()
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
localstack_main |     return self.view_functions[rule.endpoint](**req.view_args)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/utils.py", line 146, in __call__
localstack_main |     result = self.callback(request, request.url, {})
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/responses.py", line 202, in dispatch
localstack_main |     return cls()._dispatch(*args, **kwargs)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/responses.py", line 312, in _dispatch
localstack_main |     return self.call_action()
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/utils.py", line 245, in _wrapper
localstack_main |     response = f(*args, **kwargs)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/utils.py", line 277, in _wrapper
localstack_main |     response = f(*args, **kwargs)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/sqs/responses.py", line 71, in call_action
localstack_main |     status_code, headers, body = super(SQSResponse, self).call_action()
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/responses.py", line 397, in call_action
localstack_main |     response = method()
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/sqs/responses.py", line 93, in get_queue_url
localstack_main |     queue = self.sqs_backend.get_queue_url(queue_name)
localstack_main |   File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/sqs/responses.py", line 29, in sqs_backend
localstack_main |     return sqs_backends[self.region]
localstack_main | KeyError: 'fakeRegion'
localstack_main | 2020-08-26 07:13:26,409:API: 127.0.0.1 - - [26/Aug/2020 07:13:26] "POST / HTTP/1.1" 500 -
localstack_main | 2020-08-26 07:13:26,409:API: 127.0.0.1 - - [26/Aug/2020 07:13:26] "POST / HTTP/1.1" 500 -
localstack_main | 2020-08-26 07:13:26,409:API: 127.0.0.1 - - [26/Aug/2020 07:13:26] "POST / HTTP/1.1" 500 -
localstack_main | 2020-08-26 07:13:26,426:API: Error on request:
```

My docker-compose.yml 

version: '2.1'
services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
    image: localstack/localstack
    ports:
      - "4566-4599:4566-4599"
      - "${PORT_WEB_UI-8000}:${PORT_WEB_UI-8000}"
    environment:
      - SERVICES=sqs
      - DEBUG=1
      - DATA_DIR=/Users/apriya/project/local-stack/data-dir
      - PORT_WEB_UI=${PORT_WEB_UI- }
      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
      - DOCKER_HOST=unix:///var/run/docker.sock
      - HOST_TMP_FOLDER=${TMPDIR}
      - DEFAULT_REGION=us-east-1
    volumes:
      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"

**Comment by j6k4m8** [comment]
I believe you need to use a legitimate AWS region. The code in question is [here](https://github.com/spulec/moto/blob/master/moto/sqs/responses.py#L29), which calls out to [a boto3 session](https://github.com/spulec/moto/blob/master/moto/sqs/models.py#L968) that gets [the actual list of valid AWS regions for this service](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.get_available_regions).

In other words, try using `us-east-1` or another AWS region instead of `fakeRegion`.

**Comment by usmangani1** [comment]
@ankitapriya  Thanks for reporting the Issue.
Can you please help us with the sample script to reproduce the issue here.
As @j6k4m8 said the region is being interpreted as 'fakeRegion' which is not a valid aws region.
Can you please help us with any sample script using which we can reproduce the issue ?

This will help us reproduce and resolve the issue a bit sooner.Thanks

**Comment by whummer** [comment]
Thanks for providing the solutions @j6k4m8 @usmangani1 . 👍  Please report here if the problem persists when using a valid region @ankitapriya . Thanks

**Comment by ankitapriya** [comment]
Thanks @j6k4m8 @usmangani1 . I was not using the legitimate aws region
